strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/services/impl/RoutingRulesServiceImplTest.java

```diff
@@ -0,0 +1,166 @@
+package org.carlspring.strongbox.services.impl;
+
+import org.carlspring.strongbox.CommonConfig;
+import org.carlspring.strongbox.StorageApiConfig;
+import org.carlspring.strongbox.configuration.Configuration;
+import org.carlspring.strongbox.configuration.ConfigurationManager;
+import org.carlspring.strongbox.configuration.ConfigurationRepository;
+import org.carlspring.strongbox.services.ConfigurationService;
+import org.carlspring.strongbox.services.RoutingRulesService;
+import org.carlspring.strongbox.storage.routing.RoutingRule;
+import org.carlspring.strongbox.storage.routing.RuleSet;
+
+import java.util.Collections;
+import java.util.HashSet;
+
+import edu.emory.mathcs.backport.java.util.Arrays;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import static org.junit.Assert.*;
+
+/**
+ * Created by Bohdan on 8/6/2016.
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration
+public class RoutingRulesServiceImplTest
+{
+
+    private static final String RULE_PATTERN = "*.org.test";
+    private static final String REPOSITORY_ID = "repo-id";
+    public static final String GROUP_REPOSITORY = "dasdas";
```

Comment:
I know it's just a test, but could you please use something more meaningful as the name of the repository...?